cmd/earthly/main.go

```diff
@@ -951,11 +951,11 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 
 Set additional buildkit args, using a YAML array:
 
-	config global.buildkit_additional_args ['userns', '--host']
+	config global.buildkit_additional_args "['userns', '--host']"
```

Comment:
nit: how about we use single quotes for the outer yaml string, then use double quotes for the inner parts, that way it's also valid json.
```suggestion
	config global.buildkit_additional_args '["userns", "--host"]'
```